### DISCUSSION TITLE
Disable MX record checks if using Spam or other filter service in front of Addy.io

### DISCUSSION THREAD

**Discussion by evannadeau** [anonaddy/anonaddy#678] [Feature Requests]
I have a domain added to Addy.io and have been happily using it for some time. But the spam—so much spam. I'm trying out a spam filtering service in front of Addy.io, and it works great. The only problem is that you disable my domain if its records are not at the exact correct priority. Even adding the spam filter service MX records as a higher priority you still disable my domain.

The request: Allows me to remove the MX records in this scenario.  Or, allow me to set the other service MX records to a higher priority.  Something that will keep my domain operational but I have other mail services in front of addy.io.

**Comment by willbrowningme** [comment]
Please [send me an email](https://addy.io/contact/) and I can look into this for you.
